src/x/xMinIndex.ts

```diff
@@ -1,16 +1,24 @@
 import { NumberArray } from 'cheminfo-types';
 
 import { xCheck } from './xCheck';
-
+import { xGetFromToIndex, XGetFromToIndexOptions } from './xGetFromToIndex';
 /**
  * Computes the index of the minimum of the given values
  *
  * @param array - array of numbers
+ * @param options - options
```

Comment:
```suggestion
 * @param options - allows to apply the algorithm on a part of the array
```

Reply:
Are you sure about this? There may be other options in the future that are unrelated to this.

Reply:
Yes you are right. It should not be changed because we have anyway the documentation coming from the options type.